tools/tune/src/src_generate.m

```diff
@@ -242,7 +242,7 @@
 end
 
 function d = mkdir_check(d)
-if exist(d) ~= 7
+if isfolder(d) ~=7 % true if PATH points to a folder
```

Comment:
One more thing, https://se.mathworks.com/help/matlab/ref/isfolder.html

isfolder is never 7 so this should be `if ~isfolder(d)`, right?

Reply:
isfolder result is logical 0/1 but ~=7 impact is seen when Exporting the file to include folder in case ~=7 is removed
so  left ~=7 as it is
```
>> src_std_int32
Step=  1, Delta=36.133008 dB, N=131
Step=  2, Delta=34.465830 dB, N=135
Step=  3, Delta=32.693937 dB, N=139
Step=  4, Delta=30.804600 dB, N=143
Step=  5, Delta=28.776715 dB, N=147
Step=  6, Delta=26.587632 dB, N=151
Step=  7, Delta=24.200499 dB, N=155
Step=  8, Delta=21.568589 dB, N=159
Step=  9, Delta=18.614211 dB, N=163
Step= 10, Delta=15.215722 dB, N=167
Step= 11, Delta=11.139196 dB, N=171
Step= 12, Delta=5.862646 dB, N=175
Step= 13, Delta=0.263999 dB, N=179
Step= 14, Delta=0.277000 dB, N=183
Step= 15, Delta=0.276205 dB, N=187
Step= 16, Delta=0.032672 dB, N=191
Step= 17, Delta=0.107534 dB, N=195
Step= 18, Delta=-0.191142 dB, N=199
Exporting include/src_std_int32_1_2_4535_5000.h ...
error: fprintf: invalid stream number = -1
error: called from
    src_export_coef at line 43 column 2
    src_generate at line 156 column 25
    src_std_int32 at line 55 column 1
```

Reply:
Doesn't the always true if mean that mkdir() is executed always? I added this because in matlab or octave there was unnecessary warnings printed to screen if the directory existed.

Reply:
Yes mkdir() is  executed 
First when is created for first time , we don't have warning
and next time onwards ( when it already exist), we see the warnings 


```
Warning: Directory already exists. 
> In src_generate>mkdir_check (line 246)
  In src_generate (line 75)
  In src_std_int32 (line 55) 
```

I see the error due to cascaded affect as seen in below error logs  

```
Exporting include/src_std_int32_1_2_4535_5000.h ...
Error using fprintf
Invalid file identifier. Use fopen to generate a valid file identifier.

Error in src_export_coef (line 43)
	fprintf(fh, '/* SPDX-License-Identifier: BSD-3-Clause\n');

Error in src_generate (line 156)
                        src_export_coef(src1, coef_label, coef_ctype, hdir, cfg.profile);

Error in src_std_int32 (line 55)
src_generate(fs1, fs2, fs_matrix, cfg);
 
>> 

```